docs/conversion/sakai_20_mysql_conversion.sql

```diff
@@ -0,0 +1,5 @@
+-- SAK_41228
+UPDATE cm_membership_t SET USER_ID = LOWER(USER_ID);
+UPDATE cm_enrollment_t SET USER_ID = LOWER(USER_ID);
+UPDATE cm_official_instructors_t SET INSTRUCTOR_ID = LOWER(INSTRUCTOR_ID);
```

Comment:
table names should be capitalized as they were created in caps ....

`<set name="officialInstructors" table="CM_OFFICIAL_INSTRUCTORS_T" cascade="all" lazy="false">`

Reply:
Fixed it, thank you!